gradients.json

```diff
@@ -767,5 +767,11 @@
     {
         "name": "Vice City",
         "colors": ["#3494E6", "#EC6EAD"]
-    }
+    },
+    {
+        "Love and Liberty",
```

Comment:
Hey, looks like you forgot the "name" key. You can see the other gradients for reference.